broker/src/main/java/io/zeebe/broker/clustering/topology/TopologyManagerImpl.java

```diff
@@ -189,9 +189,22 @@ private BrokerInfo readBrokerInfo(final Member eventSource) {
     final BrokerInfo brokerInfo = BrokerInfo.fromProperties(eventSource.properties());
     if (brokerInfo != null && !isStaticConfigValid(brokerInfo)) {
       LOG.error(
-          "Static configuration of node {} differs from local node {}",
+          "Static configuration of node {} differs from local node {}.",
           eventSource.id(),
           atomix.getMembershipService().getLocalMember().id());
+
+          LOG.error(
+            "Static Configuration :: Local Node Configuration \n" 
```

Comment:
I like that you spend effort on making the entire statement human readable such that we can easily find the diff, but multi error log statements are generally not well received by error tooling. It would be great if you could find a way to get everything on one line, but still in a human readable format in which we can easily find the diff.

Reply:
Made some changes. Let me know what you think.